yarn.lock

```diff
@@ -13365,51 +13365,6 @@ metro-minify-uglify@0.76.7:
   dependencies:
     uglify-es "^3.1.9"
 
-metro-react-native-babel-preset@0.76.5:
```

Comment:
this is essentially the 50 lines added in the diff from https://github.com/expo/expo/pull/23476/files#diff-51e4f558fae534656963876761c95b83b6ef5da5103c4adef6768219ed76c2deR13413-R13456